crates/quickjs-wasm-rs/src/js_binding/context.rs

```diff
@@ -51,6 +53,19 @@ impl Context {
         Value::new(self.inner, raw)
     }
 
+    pub fn execute_pending(&self) -> Result<()> {
+        let runtime = unsafe { JS_GetRuntime(self.inner) };
+
+        loop {
+            let mut ctx = ptr::null_mut();
```

Comment:
AFAICT, there is no documentation anywhere regarding `JS_ExecutePendingJob`, so I can only answer based on what I'm seeing in the implementation also: yes it seems to be the context of the last resolved job.  And since Javy only ever creates a single context, presumably that's the one you'd always end up with here.

In any case, this seems to be the only way to "await" the completion of top-level `Promise`s from Rust, e.g. after calling an `async` JS function from Rust.